### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by zouhir**
Pulled the bits that are relevant to the JavaScript API from the [MSEdge explainers repo](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/Foldables/explainer.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,93 @@
...
+
+### Complementary existing proposals:
+Before discussing the solution proposal - let's overview existing proposals that are relevant and applicable to the problem space. 
+As matter of principle we should generally avoid creating redundant concepts if we can reuse existing platform APIs and capabilities.
+
+- [Presentation API](https://w3c.github.io/presentation-api/) is solving the problem of a effective use of a _secondary_ screen and likely won't attempt to solve challenges outlined above that are specific to devices where a window can span separate physical displays. This would likely still be a separate problem for foldables
+
+- [Screen Enumeration API Explainer](https://github.com/spark008/screen-enumeration/blob/master/EXPLAINER.md) provides information about the physical screen configuration. Web developers might be able to leverage that on foldables, but would need to do extra effort to correlate that information with window parameters. Some concrete examples on why a special purpose API might be useful in addition to Screen Enumeration:
```
Comment (anssiko) [inline README.md]: https://github.com/webscreens/screen-enumeration/blob/master/EXPLAINER.md

File: README.md [hunk 2]
```diff
@@ -1,3 +1,93 @@
...
+	- Getting adjacency information about spanning window regions to lay out content in several areas in logical way for a device;
+	- Getting inner window dimensions that account for application frame, OS UI elements, etc.
+- [Window Placement API Explainer](https://github.com/spark008/window-placement/blob/master/EXPLAINER.md) is useful in multi-window scenarios on multiple screen devices, but does not target scenarios in which the hosting application (i.e. browser) has a single window which spans multiple displays. In this case, the developer may not wish to open new windows - just hints to help lay out things properly and take advantage of the physical partitioning of the available layout space.
```
Comment (anssiko) [inline README.md]: https://github.com/webscreens/window-placement/blob/master/EXPLAINER.md

File: README.md [hunk 3]
```diff
@@ -1,3 +1,93 @@
...
+
+This proposal is primarily aimed at reactive scenarios, where an application wants to take advantage of the fact that it spans multiple displays, by virtue of the user/window manager placing it in that state. It is not designed for scenarios of proactively placing content in a separate top-level browsing context on the various displays available (this would fall under the [Window Placement API](https://github.com/spark008/window-placement/blob/master/EXPLAINER.md) or [Presentation API](https://w3c.github.io/presentation-api/)). Note that given the [Screen Enumeration API](https://github.com/spark008/screen-enumeration/blob/master/EXPLAINER.md) and existing primitives on the Web, it is possible to write JavaScript code that intersects the rectangles of the Display and window, while taking into account devicePixelRatio in order to compute the interesting layout regions of a window spanned across displays. However this may not correctly handle corner cases of future device form factors, and thus this proposal tries to centralize access to "here are the interesting parts of the screen a developer can target or consider for presenting content" as a practical starting point.
```
Comment (anssiko) [inline README.md]: https://github.com/webscreens/window-placement/blob/master/EXPLAINER.md
https://github.com/webscreens/screen-enumeration/blob/master/EXPLAINER.md

**Review by anssiko — APPROVED** [review]
Thanks!

Nit: the general convention is to name the explainer doc `explainer.md` (or `EXPLAINER.md`).

File: README.md (hunk 1 above)
Comment (zouhir) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (zouhir) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (zouhir) [reply README.md]: fixed

**Review by anssiko — APPROVED** [review]

**Comment by anssiko** [comment]
@zouhir you may now set a pointer (a simple link will do) from https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/Foldables/explainer.md#proposal-window-segments-enumeration-javascript-api into this repo.

**Pull request merged**